.go-version

```diff
@@ -1 +1 @@
-1.22.6
```

Comment:
We did this 3 weeks ago... https://github.com/kubernetes/kubernetes/commit/a732fb11673778368ff69dfe37092382ecc1f311

Reply:
in general, updating go versions is a long process FYI - https://github.com/kubernetes/release/issues/3738